modal/shared/logging.py

```diff
@@ -0,0 +1,107 @@
+import json
+import logging
+import os
+import sys
+
+from datadog_api_client import ApiClient, Configuration
+from datadog_api_client.v2.api.logs_api import LogsApi
+from datadog_api_client.v2.model.content_encoding import ContentEncoding
+from datadog_api_client.v2.model.http_log import HTTPLog
+from datadog_api_client.v2.model.http_log_item import HTTPLogItem
+
+
+def get_logger(name: str):
+    return logging.getLogger(name)
+
+
+# Define a custom logging handler that sends logs to Datadog
+class DatadogHandler(logging.Handler):
+    def __init__(self, api_client: ApiClient):
+        self.api_client = api_client
+
+        super().__init__()
+
+    def emit(self, record):
+        # Ignore debug messages
+        if record.levelno == logging.DEBUG:
+            return
+
+        environment_name = os.getenv("DD_ENV") or "development"
+
+        log_payload = {
+            "python-logging": {
+                "py-env": "development",
```

Comment:
Is this the case even for prod instance?

Reply:
good catch, fixed!